packages/office-ui-fabric-react/src/components/TextField/TextField.doc.tsx

```diff
@@ -90,11 +90,11 @@ export const TextFieldPageProps: IDocPageProps = {
       view: <TextFieldStyledExample />
     }
   ],
-  overview: require<string>('!raw-loader!office-ui-fabric-react/src/components/TextField/docs/TextFieldOverview.md'),
-  bestPractices: '',
-  dos: require<string>('!raw-loader!office-ui-fabric-react/src/components/TextField/docs/TextFieldDos.md'),
-  donts: require<string>('!raw-loader!office-ui-fabric-react/src/components/TextField/docs/TextFieldDonts.md'),
-  isHeaderVisible: true,
+  // overview: require<string>('!raw-loader!office-ui-fabric-react/src/components/TextField/docs/TextFieldOverview.md'),
```

Comment:
Intentional?  Can these be removed instead of commented?

Reply:
Oh oops, that was for testing...I'll undo this.